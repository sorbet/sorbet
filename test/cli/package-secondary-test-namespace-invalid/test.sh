cd test/cli/package-secondary-test-namespace-invalid || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages --secondary-test-package-namespaces=Critic:: --stripe-mode . 2>&1
../../../main/sorbet --silence-dev-message --stripe-packages --secondary-test-package-namespaces=critic --stripe-mode . 2>&1
../../../main/sorbet --silence-dev-message --stripe-packages --secondary-test-package-namespaces=^ --stripe-mode . 2>&1
