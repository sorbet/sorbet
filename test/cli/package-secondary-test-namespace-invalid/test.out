--secondary-test-package-namespaces must contain items that start with a capital letter and are alphanumeric.
--secondary-test-package-namespaces must contain items that start with a capital letter and are alphanumeric.
--secondary-test-package-namespaces must contain items that start with a capital letter and are alphanumeric.
